packages/react-instantsearch-core/src/connectors/connectInsights.ts

```diff
@@ -0,0 +1,77 @@
+import createConnector from '../core/createConnector';
+import { getResults } from '../core/indexUtils';
+
+type Without<T, K> = Pick<T, Exclude<keyof T, K>>;
```

Comment:
This type is useful enough to share across the codebase. It's often called `Omit`.

We can move it to another file when all the pending PRs are merged.